flexmeasures/api/v3_0/sensors.py

```diff
@@ -52,6 +53,19 @@
 post_sensor_schema = PostSensorDataSchema()
 sensors_schema = SensorSchema(many=True)
 
+DEPRECATED_FLEX_CONFIGURATION_FIELDS = [
```

Comment:
Actually, I'm collecting which fields are being used in `found_fields`. So we only have to add these headers for fields that are actually being used...

Reply:
The headers *are already* only added when any of these fields are being used.

Reply:
Oh, true, I didn't see that!